tool/ossl_bm.h

```diff
@@ -48,9 +48,13 @@ OSSL_MAKE_DELETER(BIGNUM, BN_free)
 OSSL_MAKE_DELETER(EC_KEY, EC_KEY_free)
 OSSL_MAKE_DELETER(EC_POINT, EC_POINT_free)
 OSSL_MAKE_DELETER(BN_CTX, BN_CTX_free)
-OSSL_MAKE_DELETER(EVP_MD_CTX, EVP_MD_CTX_free)
 OSSL_MAKE_DELETER(EVP_CIPHER_CTX, EVP_CIPHER_CTX_free)
+
+// OpenSSL 1.0 uses structs for EVP_MD/HMAC so they use the default deleter
```

Comment:
What do you mean by?
> OpenSSL 1.0 uses structs for EVP_MD/HMAC

We probably don't even need this pointers, since their object structure is public in OpenSSL 1.0.2. So, we can just put an object on the stack and get a pointer to it. But might be slightly easier what you have done, with less openssl-version conditional code.

Reply:
That was why I did it this way :)

Reply:
Makes sense :)